quarkus/tests/integration/src/test/resources/quickstart-realm.json

```diff
@@ -1,6 +1,6 @@
 {
     "realm": "quickstart-realm",
-    "enabled": true,
+    "enabled": ${TEST_REALM_ENABLED:true},
```

Comment:
We're not explicitly testing if the realm is enabled in the [`ImportAtStartupDistTest`](https://github.com/keycloak/keycloak/blob/e7ad9ff2d6a3ccb7e3e11a0ce8fc864b02bf04c4/quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/ImportAtStartupDistTest.java) but it should be fine as the import would fail if placeholders were disabled.